src/main/java/com/rapid7/client/dcerpc/mslsad/objects/SAMPRDomainLogOffInfo.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
`SAMPRDomainPasswordInfo` and `SAMPRDomainLogOffInfo` should be moved to the `mssamr` package

Reply:
👌